### DISCUSSION TITLE
How to rotate the google map with out shift

### DISCUSSION THREAD

**Discussion by thecco** [visgl/deck.gl#8744] [Q&A]
This article used a translator.

I am using google map.
I must implement rotate, tilt using the mouse right button.

I tried it as a drag event on googlemap.
However, map.SetTilt() map.setHeading() did not work while the mouse was moving.
Is there a way?

```
map.addListener('drag', function () {
    isRightButton = event.buttons == 2;
    if (isRightButton == true) {   
        map.setHeading(map.getHeading() + event.movementX);
        map.setTilt(map.getTilt() - event.movementY);
    }
});
```
Thank you

**Comment by Pessimistress** [comment]
Not relevant to this project.
